### PR TITLE
Isort everything

### LINKED ISSUE #990: Add isort to CI
**Is your feature request related to a problem? Please describe.**
Isort was running on "fix" mode, we need to add the following line to `travis.yaml` and then fix all issues isort finds.
```
## Check import order
- python -m isort ./monkey --check-only --settings-file ./ci_scripts/isort.cfg
```
This is a prerequisite for #974.

- [x] Create/edit a configuration for isort, make sure isort config is compatible with black config. Ensure travis uses the same configuration. (0d) - @shreyamalviya 
- [x] Edit travis config so that the build fails if isort finds any issues. (0d)
- [x] Reformat all imports with isort (0.25d) - @mssalvatore 
- [x] Run blackbox and unit tests to ensure nothing wonky happened (0d) - @mssalvatore

### PR DESCRIPTION
**Pull request by mssalvatore**
# What does this PR do? 

Resolves #990

Adds an isort configuration, isort check to travis, and runs isort v5.8.0 on all python code.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running unit tests and blackbox tests
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+[tool.isort]
+skip = "monkey/monkey_island/cc/ui"
+known_first_party = "common,infection_monkey,monkey_island"
+line_length = 100
+### for compatibility with black
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+ensure_newline_before_comments = true
```
Comment (VakarisZ) [inline pyproject.toml:14]: Based on [this](https://pycqa.github.io/isort/docs/configuration/black_compatibility/), wouldn't this be enough?
```
[tool.isort]
profile = "black"
multi_line_output = 3
```

**Review by VakarisZ — APPROVED** [review]
Approved, but take a look at my comment.

File: pyproject.toml (hunk 1 above)
Comment (shreyamalviya) [reply pyproject.toml:14]: It won't pick up our black config. It'll use a predefined one (which doesn't match ours). See [this](https://pycqa.github.io/isort/docs/configuration/profiles/).

File: monkey/monkey_island/cc/models/test_monkey.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 from monkey_island.cc.models.monkey import Monkey, MonkeyNotFoundError
 
-from .monkey_ttl import MonkeyTtl
 from ..test_common.fixtures import FixtureEnum
+from .monkey_ttl import MonkeyTtl
```
Comment (shreyamalviya) [inline monkey/monkey_island/cc/models/test_monkey.py:10]: Do we want relative imports?

Comment (mssalvatore) [reply monkey/monkey_island/cc/models/test_monkey.py:10]: No, but I think it's out of scope to fix them now.

**Review by shreyamalviya — APPROVED** [review]

**Pull request merged**